crawler/attribution/iframe-sandbox.html

```diff
@@ -0,0 +1,3 @@
+<script>
+    console.log(location.href, navigator.userAgent);
```

Comment:
Should this be postmessaging it's URL?

Reply:
good idea!

Reply:
Should we be printing out the result and sending it through 'messageFromSandbox'?

Reply:
Ignore sorry we're not printing the result elsewhere. (I think we should but it can be a follow up, as I think we should allow for flexibility of the API we're triggering also)

Reply:
Initially I wanted to pass the API name to the test cases, but it would require some text preprocessing or eval. There's no real use for it right now because this test is about the attribution algorithm, and that is supposed to not depend on specific API